roles/letsencrypt/tasks/main.yml

```diff
@@ -128,6 +129,25 @@
   notify: "{{ letsencrypt_reload_handler }}"
   when: letsencrypt_challenge_response.changed
 
+- name: Read certificates
+  command: "cat {{ item }}"
+  register: letsencrypt_certificates
+  changed_when: False
+  with_items:
+    - "{{ letsencrypt_cert_path }}"
+    - "{{ letsencrypt_chain_path }}"
```

Comment:
Instead of cat and register, you could use the `slurp` module to read the file into a variable.